inhouse_bot/common_utils/lol_api/tasks.py

```diff
@@ -0,0 +1,25 @@
+from typing import List
+import statistics
+
+from pyot.core.queue import Queue
+from pyot.models import lol
+
+
+async def get_summoner_by_name(summoner_name: str):
+    summoner = await lol.Summoner(name=summoner_name).get()
+    return summoner 
+    
```

Comment:
spacing in between functions, can we have 1 space between each

Reply:
I don't think it's worth nit picking these kinds of things. A style enforcer can decide how to format code. 2 spaces seems to be conventional for python as well.

Reply:
ok